src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -391,6 +400,115 @@ public void createMissingClass(UnsolvedClass missedClass) {
     }
   }
 
+  /**
+   * This method capitalizes a string. For example, "hello" will become "Hello".
+   *
+   * @param string the string to be capitalized
+   * @return the capitalized version of the string
+   */
+  public static String toCapital(String string) {
+    return string.substring(0, 1).toUpperCase() + string.substring(1);
+  }
+
+  /**
+   * This method checks if a string is capitalized
+   *
+   * @param string the string to be checked
+   * @return true if the string is capitalized
+   */
+  public static boolean isCapital(String string) {
+    String capitalizedString = toCapital(string);
+    return string.equals(capitalizedString);
```

Comment:
This also returns true if the input is all uppercase (e.g., `"LIKE THIS"`) or if the string starts with a non-letter character (e.g., `5-ary` would be considered capitalized by this code). Are these quirks intentional?

Reply:
Is there a library method that we could use instead? At a minimum, you could get the first character from the string and then check `Character#isUpperCase` directly rather than doing this equality test, which I think avoids most (all?) of the problems I mentioned.

Reply:
Yes, I think we can switch the codes to check the first character only.